core/src/test/java/io/confluent/kafka/schemaregistry/SASLClusterTestHarness.java

```diff
@@ -116,7 +116,7 @@ protected KafkaConfig getKafkaConfig(int brokerId) {
     injectProperties(props);
     props.setProperty("zookeeper.connection.timeout.ms", "30000");
     props.setProperty("sasl.mechanism.inter.broker.protocol", "GSSAPI");
-    props.setProperty(SaslConfigs.SASL_ENABLED_MECHANISMS, "GSSAPI");
+    props.setProperty(SaslConfigs.SASL_JAAS_CONFIG, "GSSAPI");
```

Comment:
Should this be BrokerSecurityConfigs.SASL_ENABLED_MECHANISMS_CONFIG instead?

Reply:
Ah, yeah, I didn't realize this was the broker-side config. I believe you are right.